api/src/app.module.ts

```diff
@@ -17,9 +17,13 @@ import { WidgetsModule } from '@api/modules/widgets/widgets.module';
 import { DataSourceManager } from '@api/infrastructure/data-source-manager';
 import { LoggingModule } from '@api/modules/logging/logging.module';
 import { SQLAdapter } from '@api/infrastructure/sql-adapter';
+import { TerminusModule } from '@nestjs/terminus';
+
+const NODE_ENV = process.env.NODE_ENV;
 
 @Module({
   imports: [
+    TerminusModule.forRoot({ logger: NODE_ENV === 'test' ? false : true }),
```

Comment:
Why do we need to discriminate? Alex no understand

Reply:
The Terminus library includes an underlying service with a logger. If the health check fails, it logs the error using this.logger.error. Some tests requires an error to occur.

I have tried to use `config.get<string>('NODE_ENV')` but it returns undefined.
is there a better way to achieve this?

Reply:
`
The Terminus library includes an underlying service with a logger. If the health check fails, it logs the error using this.logger.error. Some tests requires an error to occur.
`

Yep I've seen you have tests for it, nice. However, why does it matter if the terminus module logs the error or not? The [503 HTTP response](https://github.com/Vizzuality/4-growth/pull/147/files#diff-d907d2ad846e293157617071b0bc2ac29bdf08264719e6612f68f3f0c17cb756R41) should be returned regardless of the error is logged or not right?

`I have tried to use config.get<string>('NODE_ENV') but it returns undefined.
is there a better way to achieve this?
`

If the only purpose of this expression is to do something only for tests environment (It seems so), maybe we can do something like this? Don't know, it's not critical tho

![image](https://github.com/user-attachments/assets/34105828-c073-4884-b4a1-fff28bf84a02)